modules/sdk-coin-avaxp/test/unit/avaxp.ts

```diff
@@ -453,6 +453,80 @@ describe('Avaxp', function () {
       validTransaction.should.equal(true);
     });
 
+    it('should succeed to verify export transaction', async () => {
+      const txPrebuild = {
+        txHex: EXPORT_P_2_C.halfsigntxHex,
+        txInfo: {},
+      };
+      const txParams = {
+        recipients: [
+          {
+            address: EXPORT_P_2_C.pAddresses.join('~'),
```

Comment:
The export P 2 C recipients address should be a C Address.

Reply:
The recipients in export P 2 C is a C wallet. The export is target the sender addresses, so the import is do it by the same wallet. The export has the recipients address in the memo, we could check that.